src/lib/QueryType/ContentSubtreeQueryType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\QueryType;
+
+use eZ\Publish\Core\QueryType\QueryType;
+
+final class ContentSubtreeQueryType extends SubtreeQueryType implements QueryType
+{
+    public static function getName(): string
+    {
+        return 'ContentSubtree';
```

Comment:
```suggestion
        return 'EzPlatformAdminUi:ContentSubtree';
```

Query Type name must be unique, so IMHO we should add some unique prefix. Ref. https://doc.ezplatform.com/en/latest/guide/controllers/#naming-of-querytypes